server/channels/store/sqlstore/retention_policy_store.go

```diff
@@ -642,19 +642,23 @@ func (s *SqlRetentionPolicyStore) RemoveTeams(policyId string, teamIds []string)
 
 func subQueryIN(property string, query sq.SelectBuilder) sq.Sqlizer {
 	queryString, args := query.MustSql()
-	subQuery := fmt.Sprintf("%s IN (SELECT * FROM (%s) AS A)", property, queryString)
+
+	subQuery := fmt.Sprintf("%s IN (%s)", property, queryString)
 	return sq.Expr(subQuery, args...)
 }
 
 // DeleteOrphanedRows removes entries from RetentionPoliciesChannels and RetentionPoliciesTeams
 // where a channel or team no longer exists.
 func (s *SqlRetentionPolicyStore) DeleteOrphanedRows(limit int) (deleted int64, err error) {
 	// We need the extra level of nesting to deal with MySQL's locking
-	rpcSubQuery := sq.Select("ChannelId").
-		From("RetentionPoliciesChannels").
-		LeftJoin("Channels ON RetentionPoliciesChannels.ChannelId = Channels.Id").
-		Where("Channels.Id IS NULL").
-		Limit(uint64(limit))
+	rpcSubQuery := sq.Select("ChannelId").FromSelect(
+		sq.Select("ChannelId").
+			From("RetentionPoliciesChannels").
+			LeftJoin("Channels ON RetentionPoliciesChannels.ChannelId = Channels.Id").
+			Where("Channels.Id IS NULL").
+			Limit(uint64(limit)),
+		"A",
+	)
```

Comment:
Just for my own knowledge, can you explain this change?

Also, what is the `"A"` about?